Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Various systemd service fixes
 - Run dovecot in the foreground as a Type=simple service. This fixes systemd's
   complaints about the delayed pidfile creation.
 .
 - Start dovecot after network-online.target, to allow non-wildcard binds.
Bug-Debian: https://bugs.debian.org/865546
            https://bugs.debian.org/825562
Forwarded: no
Last-Update: 2017-06-25
--- a/dovecot.service.in
+++ b/dovecot.service.in
@@ -8,18 +8,18 @@
 Description=Dovecot IMAP/POP3 email server
 Documentation=man:dovecot(1)
 Documentation=http://wiki2.dovecot.org/
-After=local-fs.target network.target
+After=local-fs.target network-online.target
 
 [Service]
-Type=forking
-ExecStart=@sbindir@/dovecot
+Type=simple
+ExecStart=@sbindir@/dovecot -F
 PIDFile=@rundir@/master.pid
 ExecReload=@bindir@/doveadm reload
 ExecStop=@bindir@/doveadm stop
 PrivateTmp=true
 NonBlocking=yes
 # Enable this if your systemd is new enough to support it:
-#ProtectSystem=full
+ProtectSystem=full
 
 # You can add environment variables with e.g.:
 #Environment='CORE_OUTOFMEM=1'
